tutorials/Tutorial_NumericalModel_2D.jl

```diff
@@ -0,0 +1,136 @@
+#=
+# Creating 2D numerical model setups
+
+### Aim
+The aim of this tutorial is to show you how to create 2D numerical model setups that can be used as initial setups for other codes.
+
+=#
+
+
+#=
+### 2D Subduction setup
+
+Lets start with creating a 2D model setup in cartesian coordinates, which uses the `CartData` data structure
+=#
+using GeophysicalModelGenerator
+
+nx,nz = 512,128
+x = range(-1000,1000, nx);
+z = range(-660,0,    nz);
+Grid2D = CartData(XYZGrid(x,0,z))
+
+# Now we create an integer array that will hold the `Phases` information (which usually refers to the material or rock type in the simulation)
+Phases = zeros(Int64,size(Grid2D));
```

Comment:
```suggestion
Phases = zeros(Int64,size(Grid2D));
```
`Phases = zeros(Int64, nx, 1, nz)` I think is a bit more transparent to the reader